homeassistant/components/lastfm/sensor.py

```diff
@@ -54,15 +55,22 @@ class LastfmSensor(Entity):
 
     def __init__(self, user, lastfm_api):
         """Initialize the sensor."""
+        self._unique_id = hashlib.sha256(str(user).encode("utf-8")).hexdigest()
```

Comment:
The user has already been validated as a string, so we don't need to copy it here. Just apply the encoding etc.